x/rollapp/transfergenesis/ibc_module_canonical_channel_hack.go

```diff
@@ -20,13 +20,13 @@ TODO: this whole file is temporary
 type IBCModuleCanonicalChannelHack struct {
 	porttypes.IBCModule // next one
 	rollappKeeper       rollappkeeper.Keeper
-	channelKeeper       uibc.GetChannelClientState
```

Comment:
confusing IMO
u should expect the `channelKeeper` interface
the inner `uibc` methods can expects more precise `ChainIDFromPortChannelKeeper` interface.
which have confusing name as well for an interface

Reply:
ok, in this case that's a copy/paste but sure